go/chat/server.go

```diff
@@ -2070,7 +2087,92 @@ func (h *Server) UpdateTyping(ctx context.Context, arg chat1.UpdateTypingArg) (e
 	return nil
 }
 
+func (h *Server) checkInConv(ctx context.Context, convID chat1.ConversationID) (in bool, err error) {
+	defer h.Trace(ctx, func() error { return err }, fmt.Sprintf("checkInConv(%v)", convID))()
+
+	uid := h.G().Env.GetUID()
```

Comment:
Oh I forgot to mention, you should probably just pass `uid` in here, it would look a little better. We've already checked for `nil` by this point. I like to pull the `UID` out of env only at the entry point functions in server.go